ChangeLog.md

```diff
@@ -20,6 +20,14 @@ See docs/process.md for more on how version tagging works.
 
 2.0.25
 ------
+- By default (unless `EXIT_RUNTIME=1` is specified) emscripten programs running
+  under node will no longer call `process.exit()` on `exit()`.  Instead they
+  will simply unwind the stack and return to the event loop, much like they do
+  on the web.  In many cases the node process will then exit naturally if there
+  is nothing keeping the event loop going.  Note: Because of the way that
+  threads are implemented under node this means that multi-threaded programs now
+  requirte `EXIT_RUNTIME=1` in order to actually bring down the process on
+  `exit()` (or return from main).
```

Comment:
Is the symptom here that a multithreaded node program will hang? I wonder if we can at least add a warning to help with that. Or at least to mention debugging tips here in the changelog.

Reply:
(That is, mention in the changelog that the symptom is hanging.)

Reply:
It does not hang exactly, its just sits there with the event loop running waiting for more events to process.    Similar to if you did a `setInterval(1)` ..    This kind of makes sense since `EXIT_RUNTIME=0`.. in theory we could argue that is more correct to keep the event loop alive in this case.